### ISSUE TITLE
User consent giver doesn't show up in the admin UI

### ISSUE THREAD

**Issue by mooreds** [FusionAuth/fusionauth-issues#718]
### Description 

When I grant a user consent, I provide both the id of the user receiving consent and the user giving it, per https://fusionauth.io/docs/v1/tech/apis/consent#grant-a-user-consent

When I do so, both values are recorded in the user consent object and the user receiving consent has a new consent in the admin screen (on the user details tab, under consents) but the user giving the consent is absent.

### Steps to reproduce
1. Create two users (I added ages but don't know if that is needed)
1. Add them to a family
1. Create a consent.
1. Create an admin api key.
1. Run this curl script to give consent:

```
API_KEY=...
CONSENT_ID=...

curl -XPOST -H 'Content-type: application/json' -H "Authorization: $API_KEY" 'http://localhost:9011/api/user/consent' -d '{"userConsent": {"giverUserId" : "PUT_GIVER_USER_ID", "userId": "PUT_RECIPIENT_USER_ID", "consentId": "'$CONSENT_ID'"  }}'
```

Now look for the consent.
1. Log into the admin screen, go to the receiver's consent tab.
1. The consent shows up, but the 'given by' column is  `-`

However, if you run this curl script:

```
API_KEY=...

curl -XGET  -H "Authorization: $API_KEY" 'http://localhost:9011/api/user/consent/?userId=PUT_RECIPIENT_USER_ID'
```

You'll see the giver id is stored.

### Expected behavior

The admin UI should show the consent giver user and make it linked to that user's details page.

### Screenshots
<img width="1553" alt="Screen Shot 2020-06-30 at 5 23 39 PM" src="https://user-images.githubusercontent.com/91825/86186564-a4555b80-baf6-11ea-9553-9f7c13030035.png">

<img width="766" alt="Screen Shot 2020-06-30 at 5 24 40 PM" src="https://user-images.githubusercontent.com/91825/86186575-aa4b3c80-baf6-11ea-9908-b6e2a38526c6.png">

### Platform

FusionAuth 1.17.0

### Additional context

No errors in the javascript console. No errors in the fusionauth server log.

**Comment by LucasPMorris** [comment]
Similar / Related: #722 #1822

**Comment by andrewpai** [comment]
Closing, as this was fixed in 1.49.0.
